applications/dashboard/views/email/email-basic.tpl

```diff
@@ -6,7 +6,7 @@
 <!--<![endif]--><meta name="viewport" content="width=device-width, initial-scale=1.0">
 <!--[if (gte mso 9)|(IE)]>
     {literal}
-    <style type="text/css">
+    <style>
```

Comment:
Some email clients are not yet HTML5 compliant, so I don't feel comfortable removing this yet.